.github/workflows/reusable-build-flavor.yaml

```diff
@@ -72,19 +90,41 @@ jobs:
       - name: Build PR 🔧
         if: ${{ github.event_name == 'pull_request' }}
         env:
-          FLAVOR: ${{ inputs.flavor }}
-          IMAGE: quay.io/kairos/core-${{ inputs.flavor }}:latest
+          VERSION: latest
         run: |
-          earthly +ci --SECURITY_SCANS=false --VARIANT=core --IMAGE=$IMAGE --FLAVOR=$FLAVOR
+          # Desired artifact names:
+          # container image: quay.io/kairos/ubuntu:22.04-core-amd64-generic-v2.4.0
+          # iso: kairos-ubuntu-20.04-core-amd64-generic-v2.4.1-ipxe.iso
+          #
+          # TODO:
+          # - New branch:
+          #   - Merge alma, rocky, fedora to one family dockerfile
+          #   - Move rest of logic from Earthfile to Dockerfiles
+          #   - Create a reusable worflow that simply calls docker build passing the required build-args
+          #   - Delete the earthly base-image target and force the use of the above workflow anywhere needed
+          #   - Make all remaining earthly targets to use a pre-cooked KAIROS_IMAGE rather than calling +base-image
```

Comment:
Moved to the description here: https://github.com/kairos-io/kairos/issues/1897#issue-1932574939